src/Entity/File.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * File Entity class.
+ *
+ * @ORM\Entity(repositoryClass="App\Repository\FileRepository")
+ */
+class File extends Entity
+{
+    /**
+     * Doctrine identifier for File entity.
+     *
+     * @var integer
+     *
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
```

Comment:
Are we overriding the base class Entity's ID?

Reply:
I created the entity using bin/console make entity command, should've removed this. Thanks Mike